content/language-agnostic/encyclopedia/what-is-an-ide?.md

```diff
@@ -0,0 +1,28 @@
+<!--
+---
+Subjects:
+  - "What is an IDE?"
```

Comment:
Please use key words for this, rather than a title
From template: "Please first look to use Subjects that are already listed at author.codecademy.com/browse/categories"